pegasus/sites.v3/code.org/public/afe/submit.md.erb

```diff
@@ -3,6 +3,4 @@ title: Bring computer science to your classroom with Code.org and Amazon Future
 theme: responsive
 ---
 
-# Bring computer science to your classroom with Code.org and Amazon Future Engineer
-
 <%= view :amazon_future_engineer_eligibility %>
```

Comment:
I decided this intermediate page looks better without the heading; I've given the component its own heading in intermediate states instead.